src/react-apps/applications/runtime/src/shared/resources/party/getParties/getPartiesActions.ts

```diff
@@ -0,0 +1,36 @@
+import { Action } from 'redux';
+import { IParty } from '..';
+import * as ActionTypes from './getPartiesActionTypes';
+
+export interface IGetParties extends Action {
+  url: string;
```

Comment:
Can we move the url out of the saga? Move responsibility to the saga for finding the url :)

Reply:
Donity done.